app/javascript/hooks/mutations/admins/useDeleteUser.jsx

```diff
@@ -0,0 +1,20 @@
+import { useMutation, useQueryClient } from 'react-query';
+import { toast } from 'react-hot-toast';
+import axios from '../../../helpers/Axios';
+
+export default function useDeleteUser(userId) {
+  const queryClient = useQueryClient();
+
+  return useMutation(
+    (data) => axios.delete(`/users/${userId}.json`, data),
```

Comment:
Awesome, you will still need the `/admin` path, axios custom instance will only set the `baseUrl` to ``/api/v1`.
 =>`admin/users/${userId}.json`

Reply:
This is actually not using the admin user controller, the /admin was never there

Reply:
Oh, I think that we should create an `Admin::UsersController#destroy` API.
The `UsersController#destroy` API will only allow users to destroy there authenticated accounts in the future and not any GL users account.

Reply:
Yep